scripts/global.js

```diff
@@ -1730,3 +1730,17 @@ export const beforeUnloadListener = (evt) => {
     // Most browsers ignore this nowadays, but still, keep it 'just incase'
     return (evt.returnValue = translation.BACKUP_OR_ENCRYPT_WALLET);
 };
+
+function errorHandler(e) {
+    const message = `Unhandled exception. ${e.message || e.reason}`;
```

Comment:
Can we have a `<br>` before the msg/reason? Save a little space for longer errors, as Alerts do not wrap currently.

Reply:
Done, also added sanitizeHTML to prevent weird vulnerabilities